prow/config/config.yaml

```diff
@@ -490,3 +491,78 @@ presubmits:
         resources:
           requests:
             memory: "500Mi"
+  
+  metal3-io/hardware-classification-controller:
+  - name: gofmt
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - args:
+        - ./hack/gofmt.sh
+        command:
+        - sh
+        env:
+        - name: IS_CONTAINER
+          value: "TRUE"
+        image: registry.hub.docker.com/library/golang:1.13.7
+        imagePullPolicy: Always
+  - name: govet
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - args:
+        - ./hack/govet.sh
+        command:
+        - sh
+        env:
+        - name: IS_CONTAINER
+          value: "TRUE"
+        image: registry.hub.docker.com/library/golang:1.13.7
+        imagePullPolicy: Always
+  - name: markdownlint
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - args:
+        - ./hack/markdownlint.sh
+        command:
+        - sh
+        env:
+        - name: IS_CONTAINER
+          value: "TRUE"
+        image: registry.hub.docker.com/pipelinecomponents/markdownlint:latest
+        imagePullPolicy: Always
+  - name: shellcheck
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - args:
+        - ./hack/shellcheck.sh
+        command:
+        - sh
+        env:
+        - name: IS_CONTAINER
+          value: "TRUE"
+        image: koalaman/shellcheck-alpine:stable
+        imagePullPolicy: Always
+  - name: unit
+    always_run: true
+    decorate: true
+    branches:
+      - ^v1alpha1*$
```

Comment:
This means that it would run only for the v1alpha1* branches. My guess is that you mostly have master for now, so you should remove this condition

Reply:
okay, will remove it.